pep-0716.rst

```diff
@@ -0,0 +1,84 @@
+PEP: 716
+Title: Python 3.13 Release Schedule
+Author: Thomas Wouters <thomas@python.org>
+Status: Active
+Type: Informational
+Topic: Release
+Content-Type: text/x-rst
+Created: 26-May-2023
+Python-Version: 3.13
+
+
+Abstract
+========
+
+This document describes the development and release schedule for
+Python 3.13.  The schedule primarily concerns itself with PEP-sized
+items.
+
+.. Small features may be added up to the first beta
+   release.  Bugs may be fixed until the final release,
+   which is planned for October 2024.
+
+
+Release Manager and Crew
+========================
+
+- 3.13 Release Manager: Thomas Wouters
+- Windows installers: Steve Dower
+- Mac installers: Ned Deily
+- Documentation: Julien Palard
+
+
+Release Schedule
+================
+
+3.13.0 schedule
+---------------
+
+Note: the dates below use a 17-month development period that results
+in a 12-month release cadence between major versions, as defined by
```

Comment:
For clarity, we flipped to calling them "feature versions" in https://github.com/python/peps/pull/3174:

```suggestion
in a 12-month release cadence between feature versions, as defined by
```